routes/routes.js

```diff
@@ -1,8 +1,26 @@
 const express = require('express');
 const router = express.Router();
+const PATH = require('path');
+const bodyParser = require('body-parser');
+const route = require('./routes/routes');
+const dotenv = require('dotenv');
+const { ENGINE_METHOD_ALL } = require('constants');
+
 const app = express();
```

Comment:
LINES 9 -17 is not needed in routes.js because it's already mentioned in index.js.